packages/go_router/test/configuration_test.dart

```diff
@@ -152,7 +152,7 @@ void main() {
                               builder: _mockScreenBuilder,
                               routes: <RouteBase>[
                                 GoRoute(
-                                  path: 'b',
+                                  path: 'c',
```

Comment:
Could you add separate test for this scenario? It helps capture the API intent and helps prevent a regression from happening.

Reply:
Sounds like a better idea. I've moved it to a separate test ✅
(And move the two last tests to be inside `group('RouteConfiguration')`).